.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Out of curiosity what does this do?

Reply:
VSCode will usually just use whatever TS version it's currently at for type checking. This forces it to use the version installed in the project via package.json.